.github/workflows/tools.yaml

```diff
@@ -0,0 +1,18 @@
+name: OpenAPI Tools build
+on:
+  pull_request:
+    paths:
+      - "tools/**"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
```

Comment:
can we use the latest LTS (18) instead of 16 which is going EOL later this year?

Reply:
Updating. I did used 16 as this is what we are using right now.